src/sentry/grouping/enhancer/__init__.py

```diff
@@ -330,7 +334,13 @@ def as_dict(self):
             matchers[matcher.key] = matcher.pattern
         return {"match": matchers, "actions": [str(x) for x in self.actions]}
 
-    def get_matching_frame_actions(self, frames, platform, exception_data=None, cache=None):
+    def get_matching_frame_actions(
+        self,
+        match_frames: Sequence[dict[str, Any]],
```

Comment:
Naming it `match_frames` since we don't pass the original frames:
https://github.com/getsentry/sentry/blob/836922b73d64906eb55c63a7b5ac5289e147e17b/src/sentry/grouping/enhancer/__init__.py#L141-L150
https://github.com/getsentry/sentry/blob/836922b73d64906eb55c63a7b5ac5289e147e17b/src/sentry/grouping/enhancer/__init__.py#L156-L164